src/pocapi.hpp

```diff
@@ -46,7 +46,8 @@ enum DataType {
     OVMS_DATATYPE_DYNAMIC,
     OVMS_DATATYPE_MIXED,
     OVMS_DATATYPE_Q78,
-    OVMS_DATATYPE_BIN
+    OVMS_DATATYPE_BIN,
+    OVMS_DATATYPE_END
```

Comment:
Are we sure we want to have END in exposed api header? It blocks us from adding more precisions later